rfcs/wpt-results-analysis.md

```diff
@@ -0,0 +1,23 @@
+# RFC 106: Move wpt.fyi metrics generation into the WPT org
+
+## Summary
+
+Move https://github.com/Ecosystem-Infra/wpt-results and https://github.com/Ecosystem-Infra/wpt-results-analysis into the web-platform-tests GitHub org.
+
+## Details
+
+The following metrics on wpt.fyi are generated by wpt-results-analysis:
+
+- Browser Specific Failures
+- Compat 2021
+- Interop 2022
+
+wpt-results-analysis depends on wpt-results, a highly compressed form of all full wpt.fyi runs ever, currently ~52k stored in a ~385M Git repository. This makes it possible to load thousands (or all) runs into memory at the same time, speeding up metrics generation.
+
+## Risks
+
+wpt-results-analysis depends on nodegit, which only works on Node.js 14.
+
+## Alternatives
+
+We could merge the metrics generation code into the wpt.fyi repository, but the wpt-results would still need to be standalone.
```

Comment:
This would put us in an odd state whereby we had wpt.fyi generating CSV files, saving them in its own repo, then wpt.fyi pulling those CSV files, right?

In a sense, having wpt.fyi owning this data makes a lot of sense (it is, ultimately, just another projection of data it already owns), but doing it via CSV files in the repo feels a bit odd.

Reply:
I'd like to upload the data files to a storage bucket instead. That's complicated because of GCP rules around exporting service account keys, but we can probably make it work some way or another. It's weird that wpt.fyi fetching data from https://raw.githubusercontent.com/...